### DISCUSSION TITLE
Sunflower Land Merchandise

### DISCUSSION THREAD

**Discussion by CarrotCounter** [sunflower-land/sunflower-land#2712] [Ideas]
Opening up a new discussion to hear ideas about Sunflower Land merchandise. 

**Why?**
Sunflower Land has the most passionate community in web3. We know there is demand in the community for merchandise, and have already seen initial success with the first collection drop on our merchandise store. We are looking to enhance the experience with web3 elements. 

We have seen many ideas about merchandise as it is a great way for:
a) Players to purchase an item they desire
b) Players to help promote the project
c) Additional revenue to help support the core team and future expansions. 

We have seen that NFT projects have turned to merchandise to help develop their IP. A great example is Pudgy Penguins, where they have released a successful toy collection.

**Upcoming**
We have announced that we will be adding Bumpkin Merch Wearables to our Shopify Store. For each merchandise purchase, players will be airdropped their Bumpkin Wearable equivalent to their farm. Players can look like their Bumpkin! 

We think this is a great first step into adding web3 elements, but I would like to hear more!!

**Considerations:**
There are brilliant ideas in Discord, but to make this a productive discussion, please consider some of the following the points in your proposals:
- What type of player would purchase these items (i.e. whale players, majority, any specific countries)
- Estimated pricing, costs per item, likely revenue generated for core team
- Suggested Suppliers or Manufacturers
- Any Web3 elements to drive a better Sunflower Land experience
- Sales strategies

**Comment by DaanMich** [comment]
Not much traction here it seems like. 

As already stated in the discord, I had the following idea(s). 

# In-game mints -> merch logic:
A player wins an auction and gets a special code for a discount on the merch equivalent. 

The code could be displayed before the mint (on the you won screen) OR after the item is minted on the “minted splashscreen”. 

Alternatively merch discount codes could also be sent to the in-game mailbox. Should also be stated on the “minted splashscreen” that the code is visible there 📫 

The latter takes more coding work BUT I feel like this is the best approach. 

# Merch -> SFT logic: 
Players buy merch and input their farm ID on the buy page. This should be a required field so no one misses their airdrop. 

Alternatively there could be a redeem section in-game where players can input a code they receive after they bought merch. Same remark as the SFT -> merch logic. 

# Boosts? 
As already said by many in the discord, it’s detrimental that merch airdrops don’t provide an unfair advantage (eg. Seasonal tickets). They could give a very small boost as to not impact the economy + player sentiment (eg. +1% crop growth speed). 

I do tend to like the idea of unique and cute decorations more. Eg. Freya the fox plushie, unique farm skins (eg. Halloween event farm skin, Christmas snowy biome skin, …). 

These “skins” could also be sold separately I feel like in-game for matic. I know I would buy the Halloween theme for sure (dark mode lfg). 

![DF194379-DC50-42E7-A4E2-5FE1C3268FF7](https://github.com/sunflower-land/sunflower-land/assets/111318826/1f74ea3d-d674-4599-9b54-dfa60968fc47)


# Which type of player? 
I feel like mostly these purchases would be aimed at a public that has the income to spend. Typically that’s mid to upper class (with outliers ofcourse). 

I think merch should be first and foremost cool and the sales will follow. 

# Merch ideas? 
1. More simplistic designs! I love just a simple design of the SFL logo (the coin) on the top left and nothing else. Also good marketing since people would ask “what’s that logo?“. Example see added image (I know you can’t use sunnyside assets but just to get a feel). 

![2683E955-D005-47A1-B7A5-0182F0A994E1](https://github.com/sunflower-land/sunflower-land/assets/111318826/0d8d1fc1-a902-4b20-ab51-cd1a9ef5f8eb)

2. Plushies! Freya the fox, OG Pablo, OG gnome,… Speaks for itself I think
